src/services/apple_music.js

```diff
@@ -182,15 +182,16 @@ class AppleMusic {
   }
 
   wrapPlaylistData(playlistObject) {
+    const attrs = playlistObject.attributes
     return {
       id: playlistObject.id,
-      uri: playlistObject.attributes.url,
-      name: playlistObject.attributes.name,
+      uri: attrs.url,
+      name: attrs.name,
       followers: null,
-      description: playlistObject.attributes.description.short,
+      description: attrs.description ? attrs.description.short : undefined,
```

Comment:
let's make this a;
```suggestion
      description: (playlistObject.attributes.description || {short: null}).short,
```

Reply:
Actually we could remove the `attrs` alias but keep the ternary condition?

Reply:
You could, but then it'll be a long line. So you can use the hack above. To avoid repeating `playlistObject.attributes.description`

Reply:
I'm waiting on this to merge this PR. If I don't get an update soon, I'll go ahead to do this myself.

Reply:
I'm not available right now, I'm going to commit this WE.

Reply:
Okay then, I'll just go ahead.